apps/projects/app/components/Content/IssueDetail/IssueDetail.js

```diff
@@ -1,49 +1,529 @@
 import React from 'react'
-import { createPortal } from 'react-dom'
-import { theme } from '@aragon/ui'
-
-import { Detail, Navigation } from './components'
-
-const modalRoot = document.querySelector('#projects')
-
-const issueDetailStyle = {
-  position: 'absolute',
-  top: '0',
-  height: '100vh',
-  zIndex: '2',
-  width: '100vw',
-  overflowX: 'hidden',
-  background: theme.mainBackground,
-}
-
-// TODO: Much better to pass only a issueId and populate and cache data from the component, but current code is only for UI layout
-export default class IssueDetail extends React.Component {
-  el = document.createElement('div')
-
-  componentDidMount() {
-    modalRoot.appendChild(this.el)
-  }
-  componentWillUnmount() {
-    modalRoot.removeChild(this.el)
-  }
-
-  render() {
-    const { onClose, issue, onReviewApplication, onRequestAssignment, onSubmitWork, onAllocateSingleBounty, onReviewWork, onUpdateBounty } = this.props
-
-    return createPortal(
-      <div style={issueDetailStyle}>
-        <Navigation onClose={onClose} />
```

Comment:
This was the previous IssueDetail that was handling the custom navigation bar, now is completely replaced - combined by it previous Detail child component (line 36 below)